service_identity/src/main/kotlin/org/thechance/service_identity/domain/entity/Permission.kt

```diff
@@ -9,7 +9,8 @@ import org.thechance.service_identity.utils.PermissionType
 
 data class Permission(
     val id: String,
-    val permissionType: PermissionType
+    val permissionType: PermissionType,
+    val isDeleted: Boolean? = null
```

Comment:
i think we shouldn't use nullable value for isDeleted field instead use default false value